.github/workflows/manylinux/test_package_i686.sh

```diff
@@ -26,12 +26,10 @@ $PIP_INTALL_COMMAND dist/*manylinux2010_i686.whl
 $PIP_INTALL_COMMAND pytest==5.4.3 nbval ipython==7.16.1
 $PYTEST_COMMAND
 
-# Test backend test data
-# onnx.checker all existing backend data
-$PYTHON_COMAND workflow_scripts/test_generated_backend.py
-# onnx.checker all generated backend data
-$PYTHON_COMAND onnx/backend/test/cmd_tools.py generate-data
-$PYTHON_COMAND workflow_scripts/test_generated_backend.py
+# Test generated backend test data
+$PYTHON_COMMAND onnx/backend/test/cmd_tools.py generate-data
```

Comment:
what is the motivation behind generating the data in the CI pipeline? why not simply test the data which is already checked in?

Reply:
For now, it can only make sure models from test generation on various platforms (Windows, Linux, Mac) pass strict onnx.checker. It does not require the models generated from different platforms to be the same as the checked-in ones. We can probably achieve that after https://github.com/onnx/onnx/pull/3855 is merged.

Reply:
I think we should wait for #3855 and then refine this PR. 
Once #3855 is merged we should use since_version to create all the test data and run a git diff check to verify there are no diffs. This will help us validate the PR is including all the test data that it needs to (based on the unit test cases added) and that the test data is refreshed (in the past we have run into cases where author's made changes to the test case and forgot to regenerate the test data. This resulted in bad test data being checked in)